src/Site.js

```diff
@@ -435,6 +456,140 @@ Site.prototype.createPage = function (config) {
   });
 };
 
+/**
+ * Converts an existing Github wiki or docs folder to a MarkBind website.
+ */
+Site.prototype.convert = function () {
+  return this.readSiteConfig()
+    .then(() => this.collectAddressablePages())
+    .then(() => this.addIndexPage())
+    .then(() => this.addAboutPage())
+    .then(() => this.addTopNavToDefaultLayout())
+    .then(() => this.addFooterToDefaultLayout())
+    .then(() => this.addSiteNavToDefaultLayout())
+    .then(() => this.addDefaultLayoutToSiteConfig())
+    .then(() => this.printBaseUrlMessage());
+};
+
+/**
+ * Copies over README.md or Home.md to default index.md if present.
+ */
+Site.prototype.addIndexPage = function () {
+  const indexPagePath = path.join(this.rootPath, INDEX_MARKDOWN_FILE);
+  const fileNames = ['README.md', 'Home.md'];
+  const filePath = fileNames.find(fileName => fs.existsSync(path.join(this.rootPath, fileName)));
+  return fs.copyAsync(path.join(this.rootPath, filePath), indexPagePath)
+    .catch(() => Promise.reject(new Error(`Failed to copy over ${filePath}`)));
+};
+
+/**
+ * Adds an about page to site if not present.
+ */
+Site.prototype.addAboutPage = function () {
+  const aboutPath = path.join(this.rootPath, ABOUT_MARKDOWN_FILE);
+  return fs.accessAsync(aboutPath)
+    .catch(() => {
+      if (fs.existsSync(aboutPath)) {
+        return Promise.resolve();
+      }
+      return fs.outputFileAsync(aboutPath, ABOUT_MARKDOWN_DEFAULT);
+    });
+};
+
+/**
+ * Adds top navigation menu to default layout of site.
+ */
+Site.prototype.addTopNavToDefaultLayout = function () {
+  const siteLayoutPath = path.join(this.rootPath, LAYOUT_FOLDER_PATH);
+  const siteLayoutHeaderDefaultPath = path.join(siteLayoutPath, LAYOUT_DEFAULT_NAME, 'header.md');
+
+  return fs.outputFileAsync(siteLayoutHeaderDefaultPath, TOP_NAV_DEFAULT);
```

Comment:
Does this mean that, if an author calls `markbind init -c` on an existing site (could be by accident), `_markbind/layout/default/header.md` will always be overridden even if it already exist?

Reply:
Yep, you're right. I'm not sure how to prevent this from happening...since the use case doesn't consider running a conversion more than once / running it on an existing MarkBind site

Reply:
Check for `site.json`?

Reply:
How about if `init` is ran with `-c` flag, we check for `site.json` first?

Reply:
Um how about just checking for the existence of `_markbind/layout/default/header.md` and not doing anything if it exist? This would be in line with how we deal with `about.md`, and our general policy of not touching existing files with our defaults for other kinds of files as well (remember that `markbind init` itself can also be run more than once).

Reply:
But `markbind init` definitely creates a `header.md` file, so this check would mean the top nav content will never be written to `header.md`.